src/cli/index.js

```diff
@@ -78,6 +79,7 @@ exports.run = (opts = {}) => {
                     set: sets,
                     grep,
                     updateRefs,
+                    runFailed,
```

Comment:
Also, perhaps there should be an option in config and env variable for that as well? I'm not sure though. Thinking about using this in CI, it would be convenient to have separate config for CI to run only failed tests by default or specify it by using env variable. Discussble